vscode-antimony/src/server/stibium/stibium/types.py

```diff
@@ -200,13 +200,25 @@ def __init__(self, range, val):
         super().__init__(range, IssueSeverity.Warning)
         self.val = val
         self.message = ("Compartment '{}' has not been initialized, using default value").format(val)
+        
+class RateRuleNotInReaction(Issue):
+     def __init__(self, range, val):
+        super().__init__(range, IssueSeverity.Wa)
+        self.val = val
+        self.message = ("Variable '{}' is also a non-fixed species defined in a reaction").format(val)

```

Comment:
we should change all .format() to fstrings.

Reply:
Is this a general thing, and is it implemented elsewhere at this point?  If so, fix this before merging.